### PR TITLE
feat(stream-tile-handler): use revocationPhaseOutSec option when verifying signatures

### PR DESCRIPTION
**Pull request by stephhuynh18**
## Description

Use revocation phase out option when verifying jws. Should we make the revocationPhaseOut default more accessible in the docs? 

## How Has This Been Tested?

Added tests

### CODE REVIEW

**Comment by stephhuynh18** [comment]
Failing tests are the polkadot tests for esm reasons

**Comment by oed** [comment]
Should we wait to merge this before we have the name change of the variable that @stbrody suggested?

File: packages/stream-tile-handler/src/__tests__/tile-document.test.ts [hunk 1]
```diff
@@ -587,6 +592,166 @@ describe('TileDocumentHandler', () => {
     delete state.metadata.unique
     expect(state).toMatchSnapshot()
   })
+
+  it('fails to apply commit if keys revoked', async () => {
```
Comment (stbrody) [inline packages/stream-tile-handler/src/__tests__/tile-document.test.ts:596]: it's pretty hard to follow what exactly this test is doing.  Can we pull out some pieces into helper functions and add some more comments to make it clearer what we're doing at each phase of this test?  Same for other tests added

Comment (stephhuynh18) [reply packages/stream-tile-handler/src/__tests__/tile-document.test.ts:596]: yes can do

File: packages/stream-tile-handler/src/tile-document-handler.ts [hunk 2]
```diff
@@ -211,7 +213,7 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
   }
 
   /**
-   * Verifies commit signature
+   * Verifies commit signature. If a revoked key is used to create the signature, the signature is valid for 24h after the revocation
```
Comment (stbrody) [inline packages/stream-tile-handler/src/tile-document-handler.ts:216]:
```suggestion
   * Verifies commit signature. If a revoked key is used to create the signature, the signature is valid for 24h after the revocation.  This is so that if an update made before the key revocation winds up getting anchored after the revocation does, we don't fail the write unnecessarily.
   * TODO: Remove or significantly shorten this grace period once anchors happen far more frequently on the network.
   * ```

**Review by stbrody — APPROVED** [review]
LGTM mode one suggestion for additional info in a comment
